.github/workflows/prod-build.yml

```diff
@@ -355,10 +379,11 @@ jobs:
 
       - name: Sync build
         if: ${{ ! vars.SKIP_BUILD }}
-        working-directory: mdn/yari
+        working-directory: mdn/fred
         run: |-
-          gsutil -q -m -h "Cache-Control: public, max-age=3600" cp -r client/build/static gs://${{ vars.GCP_BUCKET_NAME }}/main/
-          gsutil -q -m -h "Cache-Control: public, max-age=3600" rsync -cdrj html,json,txt -y "^static/" client/build gs://${{ vars.GCP_BUCKET_NAME }}/main
+          time gsutil -q -m -h "Cache-Control: public, max-age=3600" cp -r out/static gs://${{ vars.GCP_BUCKET_NAME }}/main/
+          time gsutil -q -m -h "Cache-Control: public, max-age=3600" rsync -cr gs://${{ vars.GCP_BUCKET_NAME }}/main/static/ gs://${{ vars.GCP_BUCKET_NAME }}/fred/static/
+          time gsutil -q -m -h "Cache-Control: public, max-age=3600" rsync -cdrj html,json,txt -y "^static/" out gs://${{ vars.GCP_BUCKET_NAME }}/fred
```

Comment:
Nit: At some point we might want to change to `cache-control: no-store` here, to prevent the Cloud Bucket from returning an older version (with our public-facing CDN, this can theoretically add up to a 2h cache invalidation).
 
Cf. https://github.com/mdn/fred/blob/d8693ee2ab8ca592d50278d81480aa8de39c114d/.github/workflows/_deploy.yml#L258-L259